### PR TITLE
Added fetch-icon.sh

### PR DESCRIPTION
**Pull request by ankit-kumar-dwivedi**
>  Include synfig studio's icons files to our test suite #10 

 **Topic:** Closes: #10 
And finally I reverted everything back just before that bad commit. :grinning: 
I have added the fetch-icon.sh, as I already run it once the icons are added in _sources/icon_
The workflow of fetch-icon.sh is as follow:
```
1. Check if the "icons" folder exist. If not create it
2. Check if svn is installed. If not prompt to install.
3. Delete if "images" folder already exist in tmp(from our last fetch).
4. fetch the icons from "https://github.com/synfig/synfig/tree/master/synfig-studio/images" and place them in "tmp/images"
5. Copy the freshly fetched icons to the "sources/icon folder"

```
Thank You so much for explaining everything in such detail. ♥‿♥

### CODE REVIEW

File: fetch-icon.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+CURRENT_DIR=$(pwd)
```
Comment (morevnaproject) [inline fetch-icon.sh]: @ankit-kumar-dwivedi I suggest to replace that with this:
```CURRENT_DIR=$(cd `dirname "$0"`; pwd)```

In this case the script will work even if we launch it from outside of repository directory. ^__^

**Comment by morevnaproject** [comment]
Great, the script looks fine now! I have just one suggestion for improvement (please check my comment above). ^__^

File: fetch-icon.sh (hunk 1 above)
Comment (ankit-kumar-dwivedi) [reply fetch-icon.sh]: Done !

**Comment by ankit-kumar-dwivedi** [comment]
> Great, the script looks fine now! I have just one suggestion for improvement (please check my comment above). ^__^

@morevnaproject done !

**Comment by morevnaproject** [comment]
Awesome, thank you!

**Pull request merged**